admin/admin-default-term.php

```diff
@@ -0,0 +1,268 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Manages filters and actions related to default terms.
+ *
+ * @since 3.1
+ */
+class PLL_Admin_Default_Term {
+
+	/**
+	 * A reference to the PLL_Model instance.
+	 *
+	 * @var PLL_Model
+	 */
+	protected $model;
+
+	/**
+	 * Preferred language to assign to new contents.
+	 *
+	 * @var PLL_Language
+	 */
+	protected $pref_lang;
+
+	/**
+	 * Reference to Polylang options array
+	 *
+	 * @var array
+	 */
+	protected $options;
+
+	/**
+	 * Constructor: setups properties.
+	 *
+	 * @since 3.1
+	 *
+	 * @param object $polylang
+	 */
+	public function __construct( &$polylang ) {
+		$this->model = &$polylang->model;
+		$this->pref_lang = &$polylang->pref_lang;
+		$this->options = &$polylang->options;
+	}
+
+	/**
+	 * Setups filters and actions needed.
+	 *
+	 * @since 3.1
+	 *
+	 * @return void
+	 */
+	public function add_hooks() {
+		$taxonomies = get_taxonomies();
```

Comment:
Should we store the registered taxonomies in a class property or call this method each time we need it ?